ow_plexil/src/plans/FaultClearDemo.plp

```diff
@@ -16,9 +16,8 @@
 
 // To test the antenna, try 'rosrun ow_lander pan.py 100'.
 
-// Note that goal errors are associated only with ROS actions, and not
-// other means of controlling lander components such as ad hoc message
-// publishing.
+// Note that goal errors are triggered only by ROS actions, and not by
+// other means of lander control such as ad hoc message publishing.
```

Comment:
I think the camera trigger message might stand alone as the only way you can ever call a lander operation using a message, aside from publishing an action goal message of course, but that's really just calling an action without a client.

Reply:
Pan/Tilt can also be commanded with just messages.